src/NuGet.Core/NuGet.Protocol/FactoryExtensionsV3.cs

```diff
@@ -20,6 +20,11 @@ public static SourceRepository GetCoreV3(this Repository.RepositoryFactory facto
             return Repository.CreateSource(Repository.Provider.GetCoreV3(), source);
         }
 
+        public static SourceRepository GetCoreV3(this Repository.RepositoryFactory factory, PackageSource source)
```

Comment:
This was added in a different PR, so the changeset will disappear once you rebase :)

Reply:
Strange but this pull request says that there are no conflicts with upstream

Reply:
Weird. Did you rebase?

I'd expect git to handle this cleanly when we merge and this file will likely end up being a no-op. 

Resolving this convo as it's largely irrelevant.